tests/generation/test_generate.py

```diff
@@ -59,3 +59,23 @@ def test_model_generation_input_dimensions(neuron_model_path):
     # Using an incompatible input length
     with pytest.raises(ValueError, match="The input sequence length"):
         _test_model_generation(model, tokenizer, model.batch_size, input_length=model.max_length * 2)
+
+
+@is_inferentia_test
+@requires_neuronx
+def test_seq2seq_generation_beam(neuron_seq2seq_path):
+    model = NeuronModelForSeq2SeqLM.from_pretrained(neuron_seq2seq_path)
+    tokenizer = AutoTokenizer.from_pretrained(neuron_seq2seq_path)
+    inputs = tokenizer("translate English to German: Lets eat good food.", return_tensors="pt")
+    output = model.generate(**inputs, num_return_sequences=1)
```

Comment:
You don't pass any generation args here: what kind of parameters are actually supported ? We should test at least the most typical ones.

Reply:
More specifically, we should test for both greedy search and beam_search:
- `max_length`,
- `min_length`,
- `max_new_tokens`.

For beam_search, there are `early_stopping`, `num_beams_group/diversity_penalty`, `length_penalty` but I am not sure we support them.

I think that all other parameters like `repetition_penalty`, `bad word ids` would end up in producing logits processors, so this is not supported.